doc/topics/style-guide.md

```diff
@@ -147,6 +147,56 @@ Not covered by C++ Core Guidelines.
 
 * Test files should be prefixed with `Test`, e.g. `TestModel` instead of `ModelTests`.
 
+## ✏️ Documentation
+
+Not covered by C++ Core Guidelines.
+
+Our documentation is generated by Doxygen before being published [online](https://cesium.com/learn/cesium-native/ref-doc). Therefore, any public API should be documented with Doxygen-compatible comments that use of the following tags (in the order listed):
+
+- `@brief`: Summarize the purpose of the class, function, etc.
+- `@details`: Use for lengthier explanations, after `@brief`.
+- `@tparam`: Describe template parameters.
+- `@param`: Denote and describe function parameters.
+- `@returns`: Describe the return value of a non-`void` function.
```

Comment:
I sometimes see `@return` (singular). I like the plural version more.

Reply:
I think it's still singular, but simple present, as in "This function return**s** a this-and-that value". That nit being picked 😁 : It seems like tools generally cannot agree here. TypeScript uses `@returns`. Java uses `@return`. Doxygen supports both. No strong preference, as long as it's not `\return...`.